### DISCUSSION TITLE
HomeKit2.0 compatibility

### DISCUSSION THREAD

**Discussion by StefanS200** [rdmtc/RedMatic#561] [General]
With ios16.2 Apple is introducing honekit2.0 architecture. 

Is RedMatic Bridge compatible with iOS 16.2? Any experiences to share before I update to new 2.0 architecture?

many thanks

**Comment by hobbyquaker** [comment]
RedMatic-HomeKit 4.0 (in Entwicklung, für RedMatic 9) basiert auf aktuellem HAP-NodeJS; Fragen dazu bitte unter https://github.com/rdmtc/RedMatic-HomeKit. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
